### ISSUE TITLE
Tag/Chip does not wrap text 

### ISSUE THREAD

**Issue by shadoath** [prenda-school/design-system#415]
Seems to be a [MUI issue](https://github.com/mui/material-ui/issues/15185).

**Comment by alecdanz** [comment]
I don't think that we need this in core functionality - curious if there is a use case that you feel this would be helpful for @shadoath

**Comment by WilliamKelley** [comment]
Yes, not an issue but working as intended. You are free to use style utilities to create customizations like wrapping/multiline tags.

As Alec said, your usecase / a screenshot is always helpful for context for design and engineering -- whether it should even appear to be the case.

If it's useful, we can consider adding something like a `maxWidth` prop and utility class in core.

**Comment by slingner** [comment]
For context, we encountered some very long statuses on our Student Cards in the portal, pushing the width of the mobile page:
![image](https://user-images.githubusercontent.com/56003456/171744713-5d44c5c2-3e17-46d1-ab83-c26e527400ae.png)

**Comment by slingner** [comment]
A maxWidth prop could be helpful, although this may just be a one-off situation?

**Comment by slingner** [comment]
Here's a temp fix for this issue in portal: https://github.com/prenda-school/enrollments/pull/489

**Comment by WilliamKelley** [comment]
So a few pieces of feedback:

- leading icons are out-of-spec for Tag
- it's kinda weird you're making them look like a Section Message
- your tag label should not be a full sentence -- that's essentially a message, and you really should be using a Section Message instead. A section message is a type of **alert**, it gets the prop `role='alert'` and alerts the user about something or to take action.
    - If we bar having full sentences, this issue doesn't arise like this.
- It's weird that your content ever overflows horizontally on mobile, that indicates some other width/maxwidth problems on your containing elements.

**Comment by alecdanz** [comment]
I'm going to talk with @cynthiapoon a little on this and see if we can't adjust the design a bit.
